pkg/config/funcs/doc.go

```diff
@@ -50,6 +50,7 @@ func Docs() map[string]string {
 		"range":           "generate a list of numbers",
 		"regex":           "match the given string against the given regular expression pattern, returning captures if defined",
 		"regexall":        "same as regex but look for all matches of the given pattern rather than just the first",
+		"replace":         "`replace` searches a given string for another given substring, and replaces each occurrence with a given replacement string.",
```

Comment:
Small nitpick (sorry!) but I think this reads clearer and matches the other doc func strings.

```suggestion
		"replace":         "searches a given string for another given substring, and replaces each occurrence with a given replacement string.",
```